src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/HoverService.cs

```diff
@@ -52,8 +52,10 @@ internal sealed partial class HoverService(
             return null;
         }
 
+        var options = HoverDisplayOptions.From(_clientCapabilitiesService.ClientCapabilities);
```

Comment:
For a service, it feels like `options` should be a parameter to the method, but I suspect it will eventually get there so this comment is hopefully irrelevant (i'm reviewing commit-at-a-time)